docs/source/posts/2024/2024-05-28-week-0-robin.rst

```diff
@@ -0,0 +1,89 @@
+Week 0: Community Bonding!
+==========================
+
+.. post:: May 29 2024
+   :author: Robin Roy
+   :tags: google
+   :category: gsoc
+
+Hi, I'm Robin and I'm a 2nd year CS undergrad from Vellore Institute of Technology, Chennai. During GSoC '24 my work will be to build an LLM chatbot which will help the community by answering their questions.
+Scientific visualization is often complicated and hard for people to get used to - "Although 3D visualization technologies are advancing quickly, their sophistication and focus on non-scientific domains makes it hard for researchers to use
```

Comment:
please, add an empty line, it is dense. 

<img width="732" alt="image" src="https://github.com/fury-gl/fury/assets/23106443/5cc2c9ad-5487-42f1-bde3-be90e3810bbd">